src/Routes.js

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import Detail from '../src/pages/Detail/Detail';
```

Comment:
Routes.js 파일과 pages 폴더는 동일하게 src 폴더 하위에 있기 때문에, 경로 아래처럼 해주시면 됩니다.

```suggestion
import Detail from './pages/Detail/Detail';
```